code/app.js

```diff
@@ -0,0 +1,19 @@
+
+
+// Faq Accordion // 
+
+const acc = document.getElementsByClassName("accordion");
```

Comment:
Super small thing, but name `acc` is not very descriptive - when it comes to naming our variable, the most important thing is descriptiveness. It would be far better to call this variable `accordion` :)